beacon_node/network/src/subnet_service/attestation_subnets.rs

```diff
@@ -450,6 +459,10 @@ impl<T: BeaconChainTypes> AttestationService<T> {
         subnet: SubnetId,
         attestation: &Attestation<T::EthSpec>,
     ) -> bool {
+        // Proposer-only mode does not need to process attestations
+        if self.proposer_only {
+            return false;
```

Comment:
Does this only reject *unaggregated* attestations?

I suspect it does but I'm checking cause we'll still need to process *aggregated* attestations in order to follow the head.

Reply:
Yep, this is just unaggregated attestations